sydent/db/sqlitedb.py

```diff
@@ -28,7 +29,17 @@ def __init__(self, syd):
         logger.info("Using DB file %s", dbFilePath)
 
         self.db = sqlite3.connect(dbFilePath)
+        curVer = self._getSchemaVersion()
 
+        # We always run the schema files if the version is zero: either the db is
+        # completely empty and schema-less or it has the v0 schema, which is safe to
+        # replay the schema files.
+        if curVer == 0:
```

Comment:
ok, so the files on disk are the v0 schema, and new installations are made to be v1 by first running the v0 schema and then doing the upgrades?

that's fine, but saying so in the comment would help

Reply:
done